Makefile

```diff
@@ -26,6 +26,22 @@ securedrop-workstation-grsec: ## Builds Debian metapackage for Qubes Workstation
 install-deps: ## Install initial Debian packaging dependencies
 	./scripts/install-deps
 
+.PHONY: requirements
+requirements: ## Creates requirements files for the Python projects
+	./scripts/create-requirements
+	./scripts/update-requirements
+
+.PHONY: build-wheels
+build-wheels: syncwheels ## Builds the wheels and syncs to the localwheels directory
```

Comment:
at this step we need to dump the current shasums from what we just downloaded from S3 to a new sha256sums-downloaded.txt file, and then verify the sig on the existing sha256sums.txt, and diff the two files. Or we can end up signing the hash of a compromised wheel from S3